### PR TITLE
Split CI workflows into multiple files

### PR DESCRIPTION
**Pull request by swissspidy**
Makes it easier to re-run individual jobs

### CODE REVIEW

File: .github/workflows/continuous-integration-bundle-size.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: CI
```
Comment (spacedmonkey) [inline .github/workflows/continuous-integration-bundle-size.yml]:
```suggestion
name: Bundle size check
```

File: .github/workflows/continuous-integration-deploy-to-wiki.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+name: CI
```
Comment (spacedmonkey) [inline .github/workflows/continuous-integration-deploy-to-wiki.yml]:
```suggestion
name: Deploy QA artifacts
```

File: .github/workflows/continuous-integration-build.yml [hunk 3]
```diff
@@ -0,0 +1,102 @@
+name: CI
```
Comment (spacedmonkey) [inline .github/workflows/continuous-integration-build.yml]:
```suggestion
name: Build plugin
```

File: .github/workflows/continuous-integration-e2e.yml [hunk 4]
```diff
@@ -0,0 +1,94 @@
+name: CI
```
Comment (spacedmonkey) [inline .github/workflows/continuous-integration-e2e.yml]:
```suggestion
name: E2E Tests
```

File: .github/workflows/continuous-integration-lint-css.yml [hunk 5]
```diff
@@ -0,0 +1,44 @@
+name: CI
```
Comment (spacedmonkey) [inline .github/workflows/continuous-integration-lint-css.yml]:
```suggestion
name: CSS Lints
```

File: .github/workflows/continuous-integration-lint-js.yml [hunk 6]
```diff
@@ -0,0 +1,51 @@
+name: CI
```
Comment (spacedmonkey) [inline .github/workflows/continuous-integration-lint-js.yml]:
```suggestion
name: JavaScript Lints
```

File: .github/workflows/continuous-integration-lint-md.yml [hunk 7]
```diff
@@ -0,0 +1,44 @@
+name: CI
```
Comment (spacedmonkey) [inline .github/workflows/continuous-integration-lint-md.yml]:
```suggestion
name: Markdown Lints
```

File: .github/workflows/continuous-integration-lint-php.yml [hunk 8]
```diff
@@ -0,0 +1,58 @@
+name: CI
```
Comment (spacedmonkey) [inline .github/workflows/continuous-integration-lint-php.yml]:
```suggestion
name: PHP Lints
```

File: .github/workflows/continuous-integration-unit-js.yml [hunk 9]
```diff
@@ -0,0 +1,62 @@
+name: CI
```
Comment (spacedmonkey) [inline .github/workflows/continuous-integration-unit-js.yml]:
```suggestion
name: JS Unit Tests
```

File: .github/workflows/continuous-integration-unit-php.yml [hunk 10]
```diff
@@ -0,0 +1,86 @@
+name: CI
```
Comment (spacedmonkey) [inline .github/workflows/continuous-integration-unit-php.yml]:
```suggestion
name: PHP Unit Tests
```

**Review by spacedmonkey — DISMISSED** [review]
Screenshot 2020-05-15 at 11 57 51
The names of the jobs really need to changed to make it clear.

**Comment by swissspidy** [comment]
Workflow names are better on the `Checks` tab now:

Screenshot 2020-05-15 at 14 35 34

Had to change them a little bit to remove redundancy on the PR itself:

Screenshot 2020-05-15 at 14 38 03

Before:

Screenshot 2020-05-15 at 14 33 28

**Comment by spacedmonkey** [comment]
@swissspidy Why are there 9 expected checks awaiting here? Shouldn't all the tests pass?

**Comment by swissspidy** [comment]
@spacedmonkey Looks like GitHub Actions got confused because I renamed them. Now there are duplicate checks listed (old & new).

For example, `Markdown Code Style` does not exist anymore. It's called `Markdown Lints / lint-md` now (which passes).

**Comment by swissspidy** [comment]
In any case, the change is working as expected otherwise. Please re-review.

**Review by spacedmonkey — APPROVED** [review]
Looks to good to me.

**Pull request merged**